test/test_helper.py

```diff
@@ -0,0 +1,5 @@
+from subprocess import call
```

Comment:
As far as i understand unittest (at least in autodetect mode which is not used here), it scans for files named `test_*.py`. Therefore the name chosen here might not be the best one. You could just call it `helper.py` or if you are trying to avoid name clashes `fam_test_helper.py`. (this is optional)